.pre-commit-config.yaml

```diff
@@ -1,5 +1,11 @@
 repos:
 - repo: https://github.com/psf/black
-  rev: 19.3b0
+  rev: 19.10b0
   hooks:
   - id: black
+
+- repo: https://github.com/asottile/blacken-docs
+  rev: v1.3.0
+  hooks:
+  - id: blacken-docs
+    additional_dependencies: [black==19.10b0]
```

Comment:
Something is broken in this implementation - it shouldn't be the case that you have to declare a project and its dependencies, as that adds even more toil. Is that a defect of pre-commit?

Reply:
This is a question for @asottile.

Reply:
this is by design for `blacken-docs` -- it doesn't limit you to any specific version (`blacken-docs` works with any version of `black`) -- the suggestion is that you pick a version here (you could leave it unpinned if you wanted, that will work but will potentially introduce some undesired effects due to `black` being generally unstable between versions)